### PR TITLE
cherrytree: fix build

### LINKED ISSUE #72218: cherrytree is broken
**Describe the bug**
Cherrytree is broken.

**To Reproduce**
1. install cherrytree
2. try to run cherrytree
```
cherrytree                                                             ~
Traceback (most recent call last):
  File "/nix/store/n8hkmihkrvffbm0wdx2vm2cp1a3kv84i-cherrytree-0.38.8/bin/.cherrytree-wrapped", line 78, in <module>
    f_main(args)
  File "/nix/store/n8hkmihkrvffbm0wdx2vm2cp1a3kv84i-cherrytree-0.38.8/bin/.cherrytree-wrapped", line 49, in f_main
    import main
  File "/nix/store/n8hkmihkrvffbm0wdx2vm2cp1a3kv84i-cherrytree-0.38.8/share/cherrytree/modules/main.py", line 22, in <module>
    import gtk
ImportError: No module named gtk
```

**Metadata**
Please run `nix run nixpkgs.nix-info -c nix-info -m` and paste the result.
 - system: `"x86_64-linux"`
 - host os: `Linux 5.3.5, NixOS, 20.03pre196586.b943338ea58 (Markhor)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.3.1`
 - channels(root): `"nixos-20.03pre196586.b943338ea58"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos`

Maintainer information:
```yaml
# a list of nixpkgs attributes affected by the problem
attribute:
# a list of nixos modules affected by the problem
module:
cherrytree
```

### PR DESCRIPTION
**Pull request by prusnak**
###### Motivation for this change

Resolves #72218

###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @AndersonTorres

### CODE REVIEW

File: pkgs/applications/misc/cherrytree/default.nix [hunk 1]
```diff
@@ -11,21 +10,16 @@ stdenv.mkDerivation rec {
...
 
   installPhase = ''
-    python setup.py install --prefix="$out"
-
-    for file in "$out"/bin/*; do
-        wrapProgram "$file" \
-            --prefix PYTHONPATH : "$(toPythonPath $out):$PYTHONPATH"
-    done
+    runHook preInstall
+    ${python.interpreter} setup.py install --prefix=$out
```
Comment (veprbl) [inline pkgs/applications/misc/cherrytree/default.nix]: This look pretty generic. Can't we just remove this custom installPhase?

Comment (prusnak) [reply pkgs/applications/misc/cherrytree/default.nix]: addressed in e167c8b23bc0b9d83a9d76f0dff53278fc1652a3

File: pkgs/applications/misc/cherrytree/default.nix [hunk 2]
```diff
@@ -1,8 +1,7 @@
-{ stdenv, fetchurl, pythonPackages, gettext }:
+{ stdenv, fetchurl, python, gettext }:
```
Comment (veprbl) [inline pkgs/applications/misc/cherrytree/default.nix]: Can be `pythonPackages` again.

Comment (prusnak) [reply pkgs/applications/misc/cherrytree/default.nix]: Addressed in 4bf36fa8a8ffd6cecfebc5a08ef63f89da69442e

**Comment by veprbl** [comment]
@GrahamcOfBorg build cherrytree

**Comment by veprbl** [comment]
@GrahamcOfBorg build cherrytree

**Review by veprbl — APPROVED** [review]

**Comment by veprbl** [comment]
@prusnak Thank you!

**Pull request merged**